pghoard/pghoard.py

```diff
@@ -589,6 +590,23 @@ def main(args=None):
         print("pghoard: {!r} doesn't exist".format(config_path))
         return 1
 
+    if arg.validate:
+        try:
+            with open(arg.config, "r") as fp:
```

Comment:
Please use read_json_config_file. This will also allow you to remove json.loads() from the validate_config_file. You're also not testing the real thing right now AFAICT since this returns a python dictionary from json.load(fp) and you're passing it to validate_config_file which expects a file pointer.

Reply:
I see what you mean. I think I kept the `json.load` for the free JSON validation but that would have been unnecessary if I used `read_json_config_file`. Modifying this as suggested.